interpolate.cc

```diff
@@ -108,12 +110,14 @@ interpfilt_process(void)
             wpt_new->shortname = QString();
             wpt_new->description = QString();
 
+            frac = (double)(timen - time1) / (double)(wpt->creation_time.toTime_t() - time1);
```

Comment:
This probably breaks down for time < 1970 or > 2038 or something)  This problem isn't new and it's not your fault. Anyplace that a double is used to hold a time is probably slightly wrong these days. You could probably make frac a QDateTime and eliminate the toTime_t() and rely on the overloaded +/- operators to DTRT. If that doesn't fall out within a couple minutes, that's cool; I won't push back more than a bit on that.